### ISSUE TITLE
Dokcerfile build error:File: /app/assembly.xml does not exist.

### ISSUE THREAD

**Issue by xxywyc** [frohoff/ysoserial#126]
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.2-beta-5:single (make-assembly) on project ysoserial: Error reading assemblies: Error locating assembly descriptor: assembly.xml

need to add files to docker workdir

COPY assembly.xml .

**Comment by Bad3r** [comment]
i tried adding it but i still get the same error
```
WORKDIR /app

COPY --from=builder /app/target/ysoserial.jar .

COPY assembly.xml .    
```

**Comment by xxywyc** [comment]
FROM maven:3.5-jdk-8 as builder
  
WORKDIR /app

# download artifacts
COPY pom.xml .
COPY assembly.xml .                                    #add this line
RUN mvn dependency:resolve
RUN mvn verify
RUN mvn compiler:help

# build
COPY src ./src
RUN mvn clean package -DskipTests
RUN mv target/ysoserial-*all*.jar target/ysoserial.jar

FROM java:8-jdk-alpine

WORKDIR /app

COPY --from=builder /app/target/ysoserial.jar .

ENTRYPOINT ["java", "-jar", "ysoserial.jar"]

**Comment by xxywyc** [comment]
COPY pom.xml .
COPY assembly.xml .                 #add this line
RUN mvn dependency:resolve

**Comment by Bad3r** [comment]
thanks! that fixed it

**Comment by frohoff** [comment]
Fixed in #127
